alpaca/version.go

```diff
@@ -0,0 +1,38 @@
+package alpaca
+
+import (
+	"runtime/debug"
+	"strings"
+	"sync"
+)
+
+const repoName = "github.com/alpacahq/alpaca-trade-api-go"
+
+var (
+	once            = sync.Once{}
+	encodedVersions string
+)
+
+// GetVersion returns running go version and alpaca-trade-api-go version
+func GetVersion() string {
+	once.Do(func() {
+		var (
+			goVersion     string
+			moduleVersion string
+		)
+
+		buildInfo, found := debug.ReadBuildInfo()
+		if !found {
+			return
+		}
+		goVersion = buildInfo.GoVersion
+		for _, dep := range buildInfo.Deps {
+			if strings.HasPrefix(dep.Path, repoName) {
+				moduleVersion = dep.Version
+				break
+			}
+		}
+		encodedVersions = "APCA-GO/" + moduleVersion + "/" + goVersion
```

Comment:
```suggestion
		encodedVersions =  fmt."APCA-GO/" + moduleVersion + " GoRuntime/" + goVersion
```

now we are really being nitpicky :)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax

ex:
```
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko)
Chrome/51.0.2704.103 Safari/537.36
Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:47.0) Gecko/20100101 Firefox/47.0
Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion
```

Reply:
Addressed: https://github.com/alpacahq/alpaca-trade-api-go/commit/783440c543b38222ed2b864afaa78626d5f9968e